cogs/audio.py

```diff
@@ -2210,6 +2322,36 @@ def has_connect_perm(self, author, server):
                     message = escape(message, mass_mentions=True)
                     await self.bot.send_message(next_channel, message)
 
+    async def disp_now_playing(self, server, song, notifychan:int):
```

Comment:
`disp_now_playing` to `display_now_playing`
`notifychan` to `notify_channel`

clarity / naming convention > sparing a few characters